Week-1/cattyboard.css

```diff
@@ -0,0 +1,230 @@
+
+body
+{
+  font-family: 'Lato' !important;
+  overflow-y: scroll;
+  overflow-x: hidden;
+}
+.maindiv
+{
+    background-color:rgb(55, 53, 53);
+    height: 275px;
+    margin-top: -28px;
+}
+
+.headDiv
+{
+    width: 450px;
+    height: 100px;
+}
+
+.imgTop {
+  padding-top: 5px;
+  padding-bottom: 10px;
+  margin-left: 50px;
+  width: 225px;
+  margin-bottom: 50px;
+  margin-top: -81px;
+
+  }
+
+  h3
+  {
+    margin-left: 330px;
+    width: 530px;
+    padding-top: 30px;
+    color:white;
+    font-size: 27px;
+  }
+
+  .textSpan
+  {
+    color: white;
+    margin-left: 330px;
+    width: 530px;
+    display: block;
+
+  }
+
+  .menuList ul{
+    list-style: none;
+    margin-left: 273px;
+    margin-top: 75px;
+    width: 500px;
+  }
+  
+  .menuList li{
+    display: inline;
+  }
+
+  .iconSpan 
+  {
+    color: white;
+  }
+  navbar
+  {
+    color:beige;
+  }
+
+ .menu
+ {
+  position:fixed;
+  top: 0;
+  left: 0;
+  width: 25px;
+  height: 100%;
+  background: #eeeeee;
+  color: black;
+  font-size: 16px;
+  font-family: Arial, Helvetica, sans-serif;
+ }
+
+ .iconNav
+ {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 30px 0;
+  text-decoration: none;
+  transition: background 0.3s;
+ }
+
+ .imgdiv
+ {
+
+  height: 50px;
+  width: 50px;
+
+ }
+
+ .la, .las {
+  font-family: 'Lato';
+  font-weight: 900;
+}
+
+table {
+  border-collapse: collapse;
+  width: 100%;
+  margin-left: 40px;
+  margin-top: 25px;
+  counter-reset: row-num;
+
+}
+
+th, td {
+  text-align: left;
+  padding: 8px;
+}
+
+table tr
+{
+  counter-increment: row-num;
+}
+table tr td:first-child::before {
+  content: counter(row-num) ". ";
+}
+tr:nth-child(even) 
+{background-color: #f2f2f2;}
+
+button
+{
+  color: blue;
+}
+
+.footerDiv
+{
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100px;
+  background-color: #6577B3;
+  color: white;
+  text-align: center;  
+}
+
+.imgBottom {
+  padding-top: 89px;
+  padding-bottom: 10px;
+  margin-left: -1430px;
+  width: 85px;
+  margin-bottom: 50px;
+  margin-top: -81px;
+
+  }
+
+  .bottomDiv
+  {
+    color: black;
+    margin-left: 330px;
+    width: 150px;
+    display: block;
+    border-radius: 5px;
+    margin-top: 500px;
+  }
+
+  p
```

Comment:
why P element css you have given ?

Reply:
I think I tried using it before and forgot to remove the tag from css. Will check in with the change.

Reply:
Removed p element from CSS and pushed.